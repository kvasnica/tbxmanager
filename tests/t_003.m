function t_003
% tbxmanager sources add/remove

onCleanup(@(x) tbx_setupTest('done')); tbx_setupTest('start');

% full syntax, default source
tbxmanager show sources

% full syntax, custom source
tbxmanager source add http://control.ee.ethz.ch/~mpt/tbx/test/test.xml
tbxmanager show sources

% adding the same source twice
tbxmanager source add http://control.ee.ethz.ch/~mpt/tbx/test/test.xml

% short syntax, remove all sources, the default should stay
tbxmanager so re http://www.tbxmanager.com/package/index.xml
tbxmanager show sources
tbxmanager sour re http://control.ee.ethz.ch/~mpt/tbx/test/test.xml
tbxmanager show sources

% short syntax, custom source
tbxmanager so a http://control.ee.ethz.ch/~mpt/tbx/test/test.xml

% short syntax
tbxmanager sh so

end
