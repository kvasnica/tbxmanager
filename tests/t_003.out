--- tbxmanager show sources 
Active sources:

http://control.ee.ethz.ch/~mpt/tbx/ifa.xml
--- tbxmanager source add http://control.ee.ethz.ch/~mpt/tbx/test/test.xml 
--- tbxmanager show sources 
Active sources:

http://control.ee.ethz.ch/~mpt/tbx/ifa.xml
http://control.ee.ethz.ch/~mpt/tbx/test/test.xml
--- tbxmanager source add http://control.ee.ethz.ch/~mpt/tbx/test/test.xml 
Source "http://control.ee.ethz.ch/~mpt/tbx/test/test.xml" is already on the list.
--- tbxmanager so re http://control.ee.ethz.ch/~mpt/tbx/ifa.xml 
--- tbxmanager show sources 
Active sources:

http://control.ee.ethz.ch/~mpt/tbx/test/test.xml
--- tbxmanager sour re http://control.ee.ethz.ch/~mpt/tbx/test/test.xml 
--- tbxmanager show sources 
Active sources:

http://control.ee.ethz.ch/~mpt/tbx/ifa.xml
--- tbxmanager so a http://control.ee.ethz.ch/~mpt/tbx/test/test.xml 
--- tbxmanager sh so 
Active sources:

http://control.ee.ethz.ch/~mpt/tbx/ifa.xml
http://control.ee.ethz.ch/~mpt/tbx/test/test.xml
